CellProfiler_pipelines/Pipelines/NF1_analysis.cppipe

```diff
@@ -0,0 +1,303 @@
+CellProfiler Pipeline: http://www.cellprofiler.org
+Version:5
+DateRevision:424
```

Comment:
It does not match. I have not updated to v4.2.5, mainly since I don't need environment issues right now lol. I would like to update it if there is a bigger update.